### PR TITLE
Should hide the format code toggling

### PR DESCRIPTION
**Pull request by artemklishch**
BREAKING CHANGES:

*Commented option for Format code in files:
- src/pages/main/MainPage.tsx
- src/pages/main/hooks/useCmdk.ts 
- *Removed option format code from the 'Jumpstart.csv' file

Fix https://github.com/rnbwdev/rnbw/issues/771

### CODE REVIEW

File: src/pages/main/MainPage.tsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ export default function MainPage() {
     fileTree,
     activePanel,
     autoSave,
-    formatCode,
+    // formatCode, // TODO: it is temorally commented - in future it is possibly used or removed
```
Comment (atulbhatt-system32) [inline src/pages/main/MainPage.tsx:52]: just remove it, in future we can add it if we will really need it

File: src/pages/main/hooks/useCmdk.ts [hunk 2]
```diff
@@ -242,9 +242,10 @@ export const useCmdk = ({ cmdkReferenceData, importProject }: IUseCmdk) => {
     dispatch(setAutoSave(!autoSave));
   }, [autoSave]);
 
-  const onToggleFormatCode = useCallback(() => {
-    dispatch(setFormatCode(!formatCode));
-  }, [formatCode]);
+  // TODO: it is temorally commented - in future it is possibly used or removed
+  // const onToggleFormatCode = useCallback(() => {
```
Comment (atulbhatt-system32) [inline src/pages/main/hooks/useCmdk.ts:246]: Remove this even from the redux state as it is going to be on by default.

**Review by atulbhatt-system32 — CHANGES_REQUESTED** [review]

File: src/pages/main/hooks/useCmdk.ts (hunk 2 above)
Comment (artemklishch) [reply src/pages/main/hooks/useCmdk.ts]: ++, removed

File: src/pages/main/MainPage.tsx (hunk 1 above)
Comment (artemklishch) [reply src/pages/main/MainPage.tsx]: ++, removed

**Comment by artemklishch** [comment]
when i run the app here i see that there aren't styles
Screenshot from 2024-04-19 08-32-11

i didn't see it before. is that what we should fix?

**Review by atulbhatt-system32 — COMMENTED** [review]
@artemklishch the toggle should be removed even from the redux state. And anywhere the state variable `formatCode` is used should be removed. Format Code should always run.

**Comment by artemklishch** [comment]
> @artemklishch the toggle should be removed even from the redux state. And anywhere the state variable `formatCode` is used should be removed. Format Code should always run.

updated

**Review by atulbhatt-system32 — APPROVED** [review]

**Review by atulbhatt-system32 — COMMENTED** [review]
@artemklishch I think you are on main branch or you are not on latest dev branch.

Can you please get your branch updated with dev.

**Comment by atulbhatt-system32** [comment]
also I left a review in you other PR

**Comment by artemklishch** [comment]
> Try to do git fetch and then switch
i did that
Screenshot from 2024-04-19 15-34-12

**Comment by atulbhatt-system32** [comment]
> > Try to do git fetch and then switch
> > i did that

I think your origin is your repo but you need to fetch from rnbw repo.

Check with- git remote -v

**Comment by artemklishch** [comment]
> @artemklishch I think you are on main branch or you are not on latest dev branch.
> 
> Can you please get your branch updated with dev.

updated

**Comment by atulbhatt-system32** [comment]
@artemklishch instead of main please raise a PR on the dev branch.

**Comment by artemklishch** [comment]
> @artemklishch instead of main please raise a PR on the dev branch.

updated

**Review by atulbhatt-system32 — APPROVED** [review]

**Comment by atulbhatt-system32** [comment]
@artemklishch there's one functionality called duplicate nodes which is performed by selecting a node and then pressing `cmd+j` was not working can you test it please. 

And also take a latest pull on your branch from dev.

**Comment by artemklishch** [comment]
> @artemklishch there's one functionality called duplicate nodes which is performed by selecting a node and then pressing `cmd+j` was not working can you test it please.
> 
> And also take a latest pull on your branch from dev.

have you merged removing format coding to 'dev' branch? i do 'git pull' on the 'dev' branch, but nothing is pulled.

**Comment by artemklishch** [comment]
> @artemklishch there's one functionality called duplicate nodes which is performed by selecting a node and then pressing `cmd+j` was not working can you test it please.
> 
> And also take a latest pull on your branch from dev.

Could you please elaborate what to test here?

**Pull request merged**